devenv/lib/repository.py

```diff
@@ -30,3 +32,29 @@ def ensure_binroot(self) -> None:
 
     def config(self) -> ConfigParser:
         return get_config(f"{self.config_path}/config.ini")
+
+    def check_minimum_version(self) -> None:
+        version = importlib.metadata.version("sentry-devenv")
+
+        cfg = self.config()
+        try:
+            minimum_version = cfg["devenv"]["minimum_version"]
+        except KeyError:
+            return
+
+        min_major, min_minor, min_patch = map(int, minimum_version.split("."))
+        major, minor, patch = map(int, version.split("."))
+
+        if (major, minor, patch) < (min_major, min_minor, min_patch):
```

Comment:
you don't really need to split these up fwiw -- can just be `parsed_minimum_version` and `parsed_version`

Reply:
ah it's vestigial from when i was thinking about how to compare, before i realized you can just use tuples